.changeset/neat-roses-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/cdk": major
```

Comment:
Not sure if this is a breaking change?

Reply:
Ah, fair play! I guess all this is doing is adding a new class, and this won't break any existing code. So I guess this is `minor`, according to the rules of https://semver.org/